### DISCUSSION TITLE
Can I make ACA-Fork Plugin?

### DISCUSSION THREAD

**Discussion by MinecraftUserHAN1110** [Photon-GitHub/AntiCheatAddition#539] [Q&A] [answered]
I want to make anticheatadditions's fork plugin.

I need move check and better check's name.
And I need aura check

So I want to make new plugin.

Can I make new aca fork anticheat?

**Comment by Janmm14** [comment, marked as answer]
The GPLv3 license of ACA requires you that you need to make the COMPLETE SOURCE CODE AVAILABLE to everyone you distribute the compiled program/jar. 
Your fork and all changes and additions which operate with original or modified ACA code also need to be licensed as GPlv3.
For more details please read the GNU GPLv3 license file included in ACA: https://github.com/Photon-GitHub/AntiCheatAddition/raw/master/LICENSE
